src/BrowserInterop/wwwroot/scripts.js

```diff
@@ -138,6 +191,9 @@ browserInterop = new (function () {
         var res = me.getSerializableObject(data, [], serializationSpec);
         return res;
     };
+    this.callInstanceAction = function (instance, methodPath, ...args) {
```

Comment:
all the other methods use "Method", I don't think this renaming is relevant here

Reply:
but then you would have a conflict with the existing callInstanceMethod

Reply:
after reading your reason for this method we can keep it like that